DESCRIPTION

```diff
@@ -1,24 +1,15 @@
 Package: madshapR
 Type: Package
 Title: Support Technical Processes Following 'Maelstrom Research' Standards 
-Version: 1.0.2.9000
+Version: 1.0.3
 Authors@R: 
     c(person(given = "Guillaume",
              family = "Fabre",
              role = c("aut", "cre"),
              email = "guijoseph.fabre@gmail.com",
              comment = c(ORCID = "0000-0002-0124-9970")),
       person("Maelstrom-research group",
-             role=c("cph", "fnd")),
-      person(given = "Alexandre",
-             family = "Trottier",
-             role=c("ctb")),
-      person(given = "Tina",
-             family = "Wey",
-             role=c("ctb")),
-      person(given = "Samuel",
-             family = "El Bouzaïdi Tiali",
-             role=c("ctb")))
+             role=c("cph", "fnd")))
 Description: Functions to support rigorous processes in data cleaning, 
      evaluation, and documentation across datasets from different studies based 
      on Maelstrom Research guidelines. The package includes the core functions 
```

Comment:
Does the Description need to match the new description on GitHub?
The madshapR package provides functions for efficient data cleaning, evaluation, and documentation across different datasets. It was developed to support work at Maelstrom Research and includes functions to evaluate and summarize datasets and their associated data dictionaries, identify potential issues in content and structure, and prepare datasets and metadata for further processing. The key outputs provided by the functions are formatted datasets, standardized metadata, and tabular and visual summary reports.